test/cmd/generic-resources.sh

```diff
@@ -308,7 +309,7 @@ run_recursive_resources_tests() {
   kube::test::get_object_assert pods "{{range.items}}{{$id_field}}:{{end}}" 'busybox0:busybox1:'
   # Command
   output_message=$(! kubectl patch -f hack/testdata/recursive/pod -p='{"spec":{"containers":[{"name":"busybox","image":"prom/busybox"}]}}' --recursive 2>&1 "${kube_flags[@]}")
-  echo $output_message
+  echo "$output_message"
```

Comment:
Same nit

Reply:
There are several other places where we are now quoting strings that we should do this.

Reply:
@BenTheElder all Updated.

Reply:
thank you :pray: